ddr/appmetadata/icons.md

```diff
@@ -0,0 +1,23 @@
+# How we model project icons
+Project icons are an important part of BadgeHub. Without icons, the webside would look very bland and uninviting. The same applies for the app launcher.
+So for the app launcher and the badgehub website and badge app, we want app developers to add an icon to their app.
+
+Now the thing is that we also need multiple sizes of the icon, because the app launcher needs a small icon, but the website needs a larger icon.
+
+So for this reason, the icon property in the app config is an object with multiple sizes, like this:
+
+```json
+{
+  "icon": {
+    "16x16": "icon-16x16.png",
```

Comment:
16x16 is very small and not necessary for todays high resolution screens (even on mobile). I suggest to skip it.